app.py

```diff
@@ -128,11 +129,18 @@ def logout():
 Pages
 """
 
-@app.route('/')
+
 @app.route('/listing')
+@app.route('/listing/<tag>')
```

Comment:
I kind of like it, but I also wonder whether this is the way to go. Tags should probably be considered like any other search filter I guess (rating, name…). Which means query parameters might be more appropriate than routing?